appveyor.yml

```diff
@@ -9,17 +9,17 @@ cache:
   - inspectcode -> appveyor.yml
   - packages -> **\packages.config
 install:
-  - cmd: git submodule update --init --recursive
+  - cmd: git submodule update --init --recursive --depth=5

```

Comment:
Why depth of 5?

Reply:
framework may be lagging behind. this allows up to 5 commits behind head